invenio_accounts/views/__init__.py

```diff
@@ -9,5 +9,6 @@
 """Invenio-accounts views."""
 
 from .settings import blueprint
+from .rest import create_blueprint
 
-__all__ = ('blueprint', )
+__all__ = ('blueprint', 'create_blueprint')
```

Comment:
The naming here is a bit misguiding (since `create_blueprint` would be for the REST view). I'd say either rename (e.g. `create_rest_blueprint`) or remove and use the full path directly in `setup.py`